output/typescript/types.ts

```diff
@@ -11692,7 +11692,7 @@ export interface IngestUppercaseProcessor extends IngestProcessorBase {
 
 export interface IngestUrlDecodeProcessor extends IngestProcessorBase {
   field: Field
-  ignore_missing?: boolean
+  ignre_missing?: boolean
```

Comment:
@szabosteve @abdonpijpelink is this change intentional? just noticed it when spot-checking the latest JS code generation.

Reply:
@JoshMock No, it is not intentional. It's an error. Thanks for spotting it! I'll open a PR to fix it soon!